tests/default_vars.sh

```diff
@@ -449,3 +453,106 @@ export_35d_run ()
 export CNTL_DIR=""
 export LIST_FILES=""
 }
+export_datm ()
+{
+export FV3=false
+export S2S=false
+export DATM=true
+export DAYS=1
+export FHMAX=24
+export WLCLK=30
+export THRD=1
+export FHROT='0'
+
+# atm/ocn/ice resolution
+# GEFS
+export DATM_SRC="GEFS"
+export FILENAME_BASE='gefs.'
+export IATM=1536
+export JATM=768
+export ATMRES='C96'
+export OCNRES='100'
+export ICERES='1.00'
+export NX_GLB=360
+export NY_GLB=320
+
+# nems.configure
+export MED_restart_data=''
+export MEDCOMP="cmeps"
+export NEMS_CONFIGURE="nems.configure.datm.IN"
+export med_model="nems"
+export atm_model="datm"
+export ocn_model="mom6"
+export ice_model="cice6"
+export atm_petlist_bounds="0 15"
+export med_petlist_bounds="16 111"
+export ocn_petlist_bounds="112 231"
+export ice_petlist_bounds="232 255"
+export TASKS=256
+export TPN=40
+export NPROC_ICE='24'
+
+export ENS_NUM=1
+export SYEAR='2011'
+export SMONTH='10'
+export SDAY='01'
+export SHOUR='00'
+export CDATE=${SYEAR}${SMONTH}${SDAY}${SHOUR}
+
+export NFHOUT=6
+export FDIAG=6
+export DT_ATMOS='900'
+export DT_DYNAM_MOM6='1800'
+export DT_THERM_MOM6='3600'
+export CPL_SLOW=${DT_THERM_MOM6}
+export CPL_FAST=${DT_ATMOS}
+export coupling_interval_slow_sec=${CPL_SLOW}
+export coupling_interval_fast_sec=${CPL_FAST}
+
+export RESTART_N=${FHMAX}
+export CPLMODE='nems_orig_data'
+export cap_dbug_flag="0"
+export use_coldstart=".false."
+export RUNTYPE='startup'
+
+export WARM_START=.F.
+export CPL=.F.
+export CPLFLX=.F.
+export CPLWAV=.F.
+export CPLWAV2ATM=.F.
+export INPUT_NML=input.mom6.nml.IN
+export MODEL_CONFIGURE=datm_configure.IN
+export FIELD_TABLE="field_table"
+
+# MOM6 defaults; 1 degree
+export MOM_INPUT=MOM_input_template_100
+export MOM6_RESTART_SETTING='n'
+export MOM6_RIVER_RUNOFF='False'
+export FRUNOFF=""
+export CHLCLIM='"seawifs_1998-2006_smoothed_2X.nc"'
+# this must be set False for restart repro
+export MOM6_REPRO_LA='False'
+# since CPL_SLOW is set to DT_THERM, this should be always be false
+export MOM6_THERMO_SPAN='False'
+# no WW3
+export MOM6_USE_WAVES='False'
+
+# CICE6 defaults; 1 degree
+export DT_CICE=${DT_ATMOS}
+export MESHICE="mesh.mx${OCNRES}.nc"
+export CICEGRID="grid_cice_NEMS_mx${OCNRES}.nc"
+export CICEMASK="kmtu_cice_NEMS_mx${OCNRES}.nc"
+export RUNID='unknown'
+# set large; restart frequency now controlled by restart_n in nems.configure
+export DUMPFREQ='d'
+export DUMPFREQ_N=1000
+export USE_RESTART_TIME='.false.'
+export RESTART_EXT='.false'
+# setting to true will allow Frazil FW and Salt to be
+# included in fluxes sent to ocean
+export FRAZIL_FWSALT='.true.'
+# default to write CICE average history files
+export CICE_HIST_AVG='.true.'
+export RT_SUFFIX=""
```

Comment:
If RT_SUFFIX and BL_SUFFIX are not used, they should be removed.

Reply:
I think there must be a BL_SUFFIX defined because I didn't add it in the cpl_defaults and the tests work as expected. They do appear with the "_prod" suffix. I'm not sure which way we want it. @junwang-noaa --can you advise?

Reply:
Yes, we need BL_SUFFIX, we use it to differentiate "REPRO" and "PROD" test.

Reply:
But by specifically setting it empty, the baselines will not have the "_prod" appended, correct?

Reply:
My understanding is the test will have "prod" appended when setting it empty here.